File: solutions/1833-find-the-highest-altitude/solution.py
class Solution:
    def largestAltitude(self, gain: List[int]) -> int:
        a2=[]
        a2.append(0)
        a2.append(gain[0])
        
        sum=gain[0]
        for i in range(1,len(gain)):
            sum+=gain[i]
            
            a2.append(sum)
        return max(a2)
        
        
        

File: solutions/1833-find-the-highest-altitude/solution.c
int largestAltitude(int* gain, int gainSize) {
    int *a=(int*)malloc((gainSize+1)*sizeof(int));
    a[0]=0;
    for(int i=1;i<=gainSize;i++){
        a[i]=a[i-1]+gain[i-1];

    }
    int maxi=0;
    for(int i=0;i<=gainSize;i++){
        if(a[i]>maxi){
            maxi=a[i];

        }}
        return maxi;
    
}
